src/hipscat_import/catalog/arguments.py

```diff
@@ -57,16 +57,21 @@ class ImportArguments(RuntimeArguments):
     a positive number, this will be the order of all final pixels and we
     will not combine pixels according to the threshold"""
     lowest_healpix_order: int = 0
-    """the lowest possible healpix order that we will use for the final 
-    catalog partitioning. setting this higher than 0 will prevent creating
+    """when determining bins for the final partitioning, the lowest possible healpix order 
+    for resulting pixels. setting this higher than 0 will prevent creating
     partitions with a large area on the sky."""
-    highest_healpix_order: int = 7
+    highest_healpix_order: int = 10
```

Comment:
Should we change this default for LSDB's [`from_dataframe`](https://github.com/astronomy-commons/lsdb/blob/28375b73340bf41c7100ee22a6b91a1edaaba1a0/src/lsdb/loaders/dataframe/from_dataframe.py#L15) to make it consistent?

Reply:
Ugh. Yes. I'll send a separate PR.